sdk/synapse/Azure.Analytics.Synapse.Spark/src/Customization/SparkBatchOperation.cs

```diff
@@ -63,7 +65,7 @@ public override SparkBatchJob Value
                     throw _requestFailedException;
                 }
 #pragma warning restore CA1065 // Do not raise exceptions in unexpected locations
-                return _value;
+                return _response.Value;
```

Comment:
It feels strange to have both `_value` and `_response` in this class.  If we never update `_value`, what is it used for?

Reply:
I have some concerns about relying on `_response` in one place and setting it in others ... it feels like race conditions are possible here.  I've posted on the .NET Teams channel to ask for the team's latest guidance on LRO implementation - I know we've had some folks work on this.  I'll share that once I have it.

Reply:
This change is necessary because the previous logic is incorrect. `_value` is not updated anywhere. As a result, when the operation completes, `_value` still points to the one before polling.

Reply:
I agree that the change is needed to correct the previous logic.  My question is - do we still need `_value` in this file?  What is it used for?

Reply:
Let me remove it